src/parcsr_mv/HYPRE_parcsr_matrix.c

```diff
@@ -408,3 +408,43 @@ HYPRE_ParCSRMatrixMatvecT( HYPRE_Complex      alpha,
                alpha, (hypre_ParCSRMatrix *) A,
                (hypre_ParVector *) x, beta, (hypre_ParVector *) y) );
 }
+
+/*--------------------------------------------------------------------------
+ *--------------------------------------------------------------------------*/
+
+HYPRE_Int
+HYPRE_ParCSRMatrixDiagScale( HYPRE_ParCSRMatrix A,
+                             HYPRE_ParVector    left,
+                             HYPRE_ParVector    right )
+{
+   if (!A)
+   {
+      hypre_error_in_arg(1);
+      return hypre_error_flag;
+   }
+
+   return ( hypre_ParCSRMatrixDiagScale((hypre_ParCSRMatrix *) A,
+                                        (hypre_ParVector *) left,
+                                        (hypre_ParVector *) right) );
+}
+
+/*--------------------------------------------------------------------------
+ *--------------------------------------------------------------------------*/
+
+HYPRE_Int
+HYPRE_ParCSRMatrixCompScalingTagged(HYPRE_ParCSRMatrix   A,
```

Comment:
Where is this used?  Where would it be used?

Reply:
Currently this is used only in geos, let me write an example usage in the `ij` driver

Reply:
Hi Rob, I added an example usage and regression test for this feature in [830736a](https://github.com/hypre-space/hypre/pull/1289/commits/830736a065952a041b876ff076bd0b120c7d1193)